src/assets/stylesheets/client-info-dialog.scss

```diff
@@ -1,11 +1,23 @@
 @import 'shared';
 
+:local(.client-info-dialog) {
+  .dialog__box__contents__title {
```

Comment:
/shrug

Reply:
Heh, I'm exploiting our old BEM style CSS. I think I've become partial to @netpro2k's CSS structure now -- with `:local()` containers but plain `.foo-bar` sub class names. That would better allow for cases like this where we want to override child component styling. Not sure if there's a better way.